app/src/main/java/com/sun/makeupwindow/data/ProductDataSource.kt

```diff
@@ -6,6 +6,7 @@ import com.sun.makeupwindow.data.source.utils.OnDataLoadCallback
 interface ProductDataSource {
     interface Local{
         fun getProducts(callback: OnDataLoadCallback<List<Product>>)
+        fun getProductIdAfterLast(callback: OnDataLoadCallback<Int>)
         fun getItemProduct(idproduct: Int, callback: OnDataLoadCallback<Product>)
```

Comment:
```suggestion
        fun getProductId(id: Int, callback: OnDataLoadCallback<Product>)
```
cái này để tên là getProduct đc ko